convert/src/label_studio_to_badgerdoc/label_studio_to_badgerdoc_use_case.py

```diff
@@ -0,0 +1,257 @@
+
+    def get_output_pdf_path(self):
+        return f"files/{self.s3_output_file_id}/{self.s3_output_file_id}.pdf"
+
+    def get_output_annotations_path(self, importjob_id_created: int):
+        return f"annotation/{importjob_id_created}/{self.s3_output_file_id}/annotation.json"
+
+    def upload_badgerdoc_format_to_s3(self, importjob_id_created: int) -> None:
+        with tempfile.TemporaryDirectory() as tmp_dirname:
+            tmp_dirname = Path(tmp_dirname)
+
+            s3_output_tokens_path = self.get_output_tokens_path()
+
+            badgerdoc_tokens_path = tmp_dirname / Path("badgerdoc_tokens.json")
+            self.badgerdoc_format.export_tokens(badgerdoc_tokens_path)
+            self.s3_client.upload_file(
+                str(badgerdoc_tokens_path),
+                self.s3_output_bucket,
+                s3_output_tokens_path,
+            )
+
+            pdf_path = tmp_dirname / Path("badgerdoc_render.pdf")
+            self.badgerdoc_format.export_pdf(pdf_path)
+
+            s3_output_pdf_path = self.get_output_pdf_path()
+            self.s3_client.upload_file(
+                str(pdf_path), self.s3_output_bucket, s3_output_pdf_path
+            )
+
+            badgerdoc_annotations_path = tmp_dirname / Path(
+                "badgerdoc_annotations.json"
+            )
+            self.badgerdoc_format.export_annotations(
+                badgerdoc_annotations_path
+            )
+            s3_output_annotations_path = self.get_output_annotations_path(
+                importjob_id_created
+            )
+            self.s3_client.upload_file(
+                str(badgerdoc_annotations_path),
+                self.s3_output_bucket,
+                s3_output_annotations_path,
+            )
+
+    def request_jobs_to_create_importjob(self):
+        post_importjob_url = f"{settings.job_service_url}create_job/"
+        post_importjob_body = {
+            "name": f"import_job_{uuid4()}",
+            "type": "ImportJob",
+            "import_source": self.s3_input_annotation.path,
+            "import_format": "Label Studio",
+            "owners": [
+                self.token_data.user_id,
+            ],
+            "files": [],
+            "categories": [],
+        }
+        LOGGER.debug(f"{post_importjob_url=}")
+        LOGGER.debug(f"{post_importjob_body=}")
+
+        try:
+            request_to_post_importjob = requests.post(
+                url=post_importjob_url,
+                headers=self.request_headers,
+                json=post_importjob_body
+            )
+            request_to_post_importjob.raise_for_status()
+        except requests.exceptions.RequestException as e:
+            raise requests.exceptions.RequestException(
+                f"Failed request to 'jobs' - {e}"
+            ) from e
+
+        else:
+            LOGGER.debug(f"{request_to_post_importjob.json()=}")
+            return request_to_post_importjob.json()["id"]
```

Comment:
No need for else here.
And make logging message more specific, please
```suggestion
        LOGGER.debug(f"{request_to_post_importjob.json()=}")
        return request_to_post_importjob.json()["id"]
```